octopod-backend/src/Octopod/Server.hs

```diff
@@ -1214,18 +1215,20 @@ runStatusUpdater state = do
     logErr  = logWarning (logger state)
 
   forever $ do
-    rows :: [(DeploymentName, DeploymentStatus, Int)] <- liftIO $
+    rows :: [(DeploymentName, DeploymentStatus, Int, DeploymentTag)] <- liftIO $
       withResource pgPool $ \conn -> query conn selectDeps (Only interval)
     let
-      checkList :: [(DeploymentName, DeploymentStatus, Timestamp)] =
-        (\(n, s, t) -> (n, s, coerce t)) <$> rows
-    checkResult <- for checkList $ \(dName, dStatus, ts) -> do
+      checkList :: [(DeploymentName, DeploymentStatus, Timestamp, DeploymentTag)] =
+        (\(n, s, t, dTag) -> (n, s, coerce t, dTag)) <$> rows
+    checkResult <- for checkList $ \(dName, dStatus, ts, dTag) -> do
       let
         args              =
           [ "--project-name", unpack . coerce $ projectName state
           , "--base-domain", unpack . coerce $ baseDomain state
           , "--namespace", unpack . coerce $ namespace state
-          , "--name", unpack . coerce $ dName ]
+          , "--name", unpack . coerce $ dName
+          , "--tag", unpack . unDeploymentTag $ dTag
+          ]
```

Comment:
`args` are common args for `checkingCommand` and `archiveCheckingCommand`.
`checkingCommand` supports `--tag`, but `archiveCheckingCommand` doesn't support `--tag`.
IMHO, can use `checkCommandArgs` and `archiveCheckCommandArgs` (look like `checkCommandArgs` only without `--tag`) here.